docker-compose/dse-6.8/docker-compose.yml

```diff
@@ -3,9 +3,10 @@ version: '2'
 services:
   dse-1:
     image: datastax/dse-server:${DSETAG}
+    platform: linux/amd64
     networks:
       - stargate
-    mem_limit: 2G
+    mem_limit: 2560M
```

Comment:
On my M1 macbook pro, DSE nodes fail silently during startup without this memory increase. Not clear if this is due to upgrading DSE version or my machine architecture, but this doesn't harm anything.